server/routes/acctRoutes.js

```diff
@@ -4,12 +4,12 @@ import auth from '../authorization/auth';
 
 const AcctRouter=Router();
```

Comment:
Make this variable name more descriptive

Reply:
changed it to AccountRouter